File: app/src/main/java/com/app/rupiksha/models/SupportTicket.java
package com.app.rupiksha.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class SupportTicket
{
    @SerializedName("id")
    @Expose
    private int id;
    @SerializedName("uid")
    @Expose
    private String uid;
    @SerializedName("ticketid")
    @Expose
    private String ticketid;
    @SerializedName("service")
    @Expose
    private String service;
    @SerializedName("txnid")
    @Expose
    private String txnid;
    @SerializedName("message")
    @Expose
    private String message;
    @SerializedName("adminmsg")
    @Expose
    private String adminmsg;
    @SerializedName("status")
    @Expose
    private String status;
    @SerializedName("date")
    @Expose
    private String date;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getUid() {
        return uid;
    }

    public void setUid(String uid) {
        this.uid = uid;
    }

    public String getTicketid() {
        return ticketid;
    }

    public void setTicketid(String ticketid) {
        this.ticketid = ticketid;
    }

    public String getService() {
        return service;
    }

    public void setService(String service) {
        this.service = service;
    }

    public String getTxnid() {
        return txnid;
    }

    public void setTxnid(String txnid) {
        this.txnid = txnid;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getAdminmsg() {
        return adminmsg;
    }

    public void setAdminmsg(String adminmsg) {
        this.adminmsg = adminmsg;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }
}

File: app/src/main/java/com/app/rupiksha/models/SupportTicket.kt
package com.app.rupiksha.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class SupportTicket(
    @SerializedName("id")
    @Expose
    val id: Int,
    
    @SerializedName("uid")
    @Expose
    val uid: String? = null,
    
    @SerializedName("ticketid")
    @Expose
    val ticketid: String? = null,
    
    @SerializedName("service")
    @Expose
    val service: String? = null,
    
    @SerializedName("txnid")
    @Expose
    val txnid: String? = null,
    
    @SerializedName("message")
    @Expose
    val message: String? = null,
    
    @SerializedName("adminmsg")
    @Expose
    val adminmsg: String? = null,
    
    @SerializedName("status")
    @Expose
    val status: String? = null,
    
    @SerializedName("date")
    @Expose
    val date: String? = null
)
